cms/static/js/spec/views/group_configuration_spec.js

```diff
@@ -491,7 +491,9 @@ define([
             AjaxHelpers.expectNoRequests(requests);
         });
 
-        it('should have appropriate class names on focus/blur', function (done) {
+        xit('should have appropriate class names on focus/blur', function (done) {
```

Comment:
I didn't know that about jQuery deleting their own focus tests, interesting. I think you're right and this should be removed, I'll push that up now.